### ISSUE TITLE
R.length not returning NaN

### ISSUE THREAD

**Issue by mohit61** [selfrefactor/rambda#606]
Hi, 

For below objects, R.length is not returning NaN
```
console.log( R.length({length: ''}));
console.log( R.length({length: '1.23'}));
console.log( R.length({length: null}));
```
output:
```

1.23
null
```
expected:
```
NaN
NaN
NaN
```

**Comment by selfrefactor** [comment]
I did make a change so that this issue is fixed. You'll be able to use it once a new version is released. I will keep the issue open, until it happens.

This is the current test proving the fix:

```
test('with length as property', () => {
  const input1 = {length: '123'}
  const input2 = {length: null}
  const input3 = {length: ''}

  expect(length(input1)).toBeNaN()
  expect(lengthRamda(input1)).toBeNaN()
  expect(length(input2)).toBeNaN()
  expect(lengthRamda(input2)).toBeNaN()
  expect(length(input3)).toBeNaN()
  expect(lengthRamda(input3)).toBeNaN()
})

```

**Comment by selfrefactor** [comment]
I am closing the issue as the fix is released with version `7.0.0`.
